ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml

```diff
@@ -36,34 +36,21 @@ tests:
     make unit-test
   container:
     from: src
-- as: rosa-classic-e2e-presubmit
-  skip_if_only_changed: ^(LICENSE|OWNERS|README\.md|\.gitignore|\.goreleaser\.yaml)$|^docs/|^subsystem/
```

Comment:
instead of removing let's make it totally optional for running:
```
- as: rosa-classic-e2e-presubmit
  always_run: false
  ...
```

this way we can still run it if we want to test something more specific

Reply:
Run it how?  /pj-rehearse u mean?

Reply:
no, with ``/test <name>``
with ``always_run: false`` you have jobs that are not run when you push a new commit but do run if you add a comment with ``/test <their-name>``